sphinx_automodapi/automodsumm.py

```diff
@@ -294,7 +294,8 @@ def automodsumm_to_autosummary_lines(fn, app):
         # Note: we use __name__ here instead of just writing the module name in
         #       case this extension is bundled into another package
         from . import automodapi
-        if automodapi.__name__ in app._extensions:
+
+        if automodapi.__name__ in getattr(app, '_extensions', 'extensions'):
```

Comment:
I still prefer the nested version, but got voted down with it in sphinx-gallery already :) They went with 
```
# Sphinx < 1.6 calls it `_extensions`, >= 1.6 is `extensions`.
extensions_attr = '_extensions' if hasattr(app, '_extensions') else 'extensions'
if 'sphinx.ext.autodoc' in getattr(app, extensions_attr):
```

Should we do the same?